src/NextGen/ServiceProvider.php

```diff
@@ -20,7 +21,7 @@ class ServiceProvider implements ServiceProviderInterface
      */
     public function register()
     {
-        //
+        give()->singleton(DonationFormRepository::class);
```

Comment:
Can we add a `give()->forms` alias?

Reply:
2a08ce23

The main reason i've refrained from this is because the IDE isn't aware of it until we add the method doc to the main give class.  Unless you know of a convenient way, I didn't want to do this everytime. 

```
/** @var DonationFormsRepository $forms */
$forms = give()->forms;
```

Reply:
Ah, I see what you're saying. We could add a `.phpstorm.meta.php` file to Next Gen that tells PhpStorm how to handle this specific situation. But perhaps this should be used via DI until its in core? Not sure. 🤔

Reply:
I would rather wait until it's in core then start adding temporary code editor specific files.

Reply:
I don't think it's a big deal either way. Obviously DI will always work.